hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -78,8 +78,8 @@ const (
 	controlPlaneOperatorCreatesDefaultAWSSecurityGroup = "io.openshift.hypershift.control-plane-operator-creates-aws-sg"
 
 	labelManagedPrefix = "managed.hypershift.openshift.io"
-	// mirroredConfigLabel added to objects that were mirrored from the node pool namespace into the HCP namespace
-	mirroredConfigLabel = "hypershift.openshift.io/mirrored-config"
+	// MirroredConfigLabel added to objects that were mirrored from the node pool namespace into the HCP namespace
+	MirroredConfigLabel = "hypershift.openshift.io/mirrored-config"
```

Comment:
can this be NTOMirroredConfigLabel so the scope is evident for a code reader with little context

Reply:
Done.